src/main/java/uk/gov/companieshouse/filingmock/model/FilingProcessed.java

```diff
@@ -6,141 +6,255 @@
 import com.fasterxml.jackson.annotation.JsonInclude.Include;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
+/**
+ * The type Filing processed.
+ */
 @JsonInclude(Include.NON_NULL)
 @JsonAutoDetect(getterVisibility = Visibility.NONE)
 public class FilingProcessed {
 
-    @JsonInclude(Include.NON_NULL)
-    static class Presenter {
-        @JsonProperty(value = "user_id")
-        String userId;
-
-        @JsonProperty(value = "language")
-        String language;
-    }
-
     @JsonProperty(value = "transaction_id")
     private String transactionId;
-
     @JsonProperty(value = "submission_id")
     private String submissionId;
-
     @JsonProperty(value = "channel_id")
     private String channelId;
-
     @JsonProperty(value = "company_number")
     private String companyNumber;
-
     @JsonProperty(value = "company_name")
     private String companyName;
-
     @JsonProperty(value = "application_id")
     private String applicationId;
-
     @JsonProperty(value = "presenter")
     private Presenter presenter;
-
     @JsonProperty(value = "processed_at")
     private String processedAt;
-
     @JsonProperty(value = "status")
     private Status status;
-
     @JsonProperty(value = "rejection")
     private Rejection rejection;
 
+    /**
+     * Gets transaction id.
+     *
+     * @return the transaction id
+     */
     public String getTransactionId() {
```

Comment:
i'd argue against adding comments to getters and setters, but appreciate it's a subjective opinion